gitlabform/lists/projects.py

```diff
@@ -113,6 +117,39 @@ def _get_projects(self, target: str, groups: Groups) -> Projects:
                 archived_projects_from_configuration_not_from_groups,
             )
 
+        if target == "ALL":
+            # in this case we also need to get the list of projects being transfered
+            # to a different namespace from the config
+
+            projects_from_configuration = self.configuration.get_projects()
+
+            for project in projects_from_configuration:
+                try:
+                    project_transfer_source = self.configuration.config[
+                        "projects_and_groups"
+                    ][project]["project"]["transfer_from"]
+                except KeyError:
+                    # ignore projects that don't contain transfer_from key
+                    continue
+
+                if project_transfer_source:
+                    if project_transfer_source in projects_from_groups:
```

Comment:
I think we don't need the outer condition.

Reply:
This has been refactored.

Reply:
Thanks for the update.